### ISSUE TITLE
Parameter connection validation incorrect 

### ISSUE THREAD

**Issue by Korny666** [osate/osate2-core#616]
```javascript

package connection_bug
public
	with Base_Types;

	thread Test
		features
			setterAndGetter: feature group setGet;
	end Test;

	thread implementation Test.impl
		subcomponents
			Counter: data Base_Types::Integer;
		connections
			set: parameter setterAndGetter.setValue -> Counter;
			return: parameter Counter -> setterAndGetter.setValue;
	end Test.impl;

	feature group setGet
		features
			setValue: in data port Base_Types::Integer;
			getValue: out data port Base_Types::Integer;
	end setGet;
end connection_bug;

```

The connections set and return are not visible.

![bug](https://cloud.githubusercontent.com/assets/12656521/9033447/4a5432ec-39c7-11e5-9d8c-3c40351981d6.png)

**Comment by lwrage** [comment]
The AADL model is not valid. According to the 9.3 (L2) the parameter connection must have a call parameter as its destination (source)  if the source (destination) is a thread port.

**Comment by lwrage** [comment]
@philip-alldredge Should the graphical editor show invalid connections?

**Comment by philip-alldredge** [comment]
I'm out of the office this week but I'll take a look when I return.
Typically the graphical editor will show invalid connections when possible.

Philip Alldredge
Research Scientist
Rotorcraft Systems Engineering and Simulation Center
University of Alabama in Huntsville
VBRH A-4
256-824-4837
Philip.Alldredge@uah.edu

**Comment by philip-alldredge** [comment]
Fixed by osate/osate-ge 39157cb0cd717ae6090bd18e742a097eb5b27eed.

**Comment by philip-alldredge** [comment]
There was a bug affecting connections involving features inside of feature groups. I just pushed a fixed to the graphical editor. However, the listed model will not display as expected because the use of set as the connection name. set is a reserved word in AADL.

Please report any additional issues with the graphical capability directly to the graphical editor project. This ensures that it will be seen by the developers working on that project.https://github.com/osate/osate-ge/issues
